src/deepsparse/transformers/utils/decoder_kv_cache.py

```diff
@@ -103,79 +103,78 @@ def update(
             Corresponds to `input_ids.shape[1]`
         """
         self.total_num_processed_tokens += input_ids_len
-        total_cache_capacity = state[list(state.keys())[0]].shape[
+
```

Comment:
Could we also update the docstring above on line 94; there's a typo
`taking the kv cache output of from the forward pass` --> `taking the kv cache output from the forward pass`

Reply:
Good catch!